stylelint/styles.css

```
@@ -0,0 +1,72 @@
+    --secondary: #abc1234;
+    --accent: #12;
+    --muted: #d3d3d3;
+}
+
+body {
+    colour: #ffffff;
+    background: var(--primary);
+    text-style: bold;
+    margin: 0px;
+    padding: 0;
+    font-family: Arial Black, sans-serif;
```

Stylelint (17.9.0)

[error] Expected quotes around "Arial Black" (font-family-name-quotes)

(font-family-name-quotes)